### DISCUSSION TITLE
How to shows custom properties in Unreal

### DISCUSSION THREAD

**Discussion by nmfarras** [JSBSim-Team/jsbsim#1101] [Q&A] [answered]
Hi, i might missing the discussion or issues thread related to this, but as far as i know currently jsbsim plugin for Unreal only support writing value and we can see the value change from the out pin of the command or batch command.  Is it possible to simply call the custom property and got the current value without updating or writing custom property value currently? Thank you in advance

![image](https://github.com/JSBSim-Team/jsbsim/assets/30061607/700d3cb5-67f8-45eb-84a3-7549618f79cf)

**Comment by gallonmate** [comment, marked as answer]
Yes you can leave the In Value emtpy/blank and it will return only the current property value. There's a little tip comments box in the Unreal Readme page, or if you hover over the node in the editor. But I realize now the tips aren't written clearly :)

**Reply by nmfarras** [reply]
Thank you, i just realize it too.
